cfme/utils/bz.py

```diff
@@ -207,6 +209,58 @@ def resolve_blocker(self, blocker, version=None, ignore_bugs=None, force_block_s
                 return bug
         return None
 
+    def set_flags(self, idlist, flags):
+        bz_creds = credentials.get("bugzilla", None)
+
+        # login to bzapi
+        if not bz_creds:
+            raise BugzillaError("No bugzilla creds available")
+        try:
+            self.bugzilla.login(bz_creds.get("username", None), bz_creds.get("password", None))
+        except BugzillaError:
+            logger.exception("Unable to login to Bugzilla with those creds.")
+            raise
+
+        # set the flags
+        for bug in self.bugzilla.getbugs(idlist):
+            result = bug.updateflags(flags)
+            logger.info("Got %s from updating %s", result, bug)
+
+        # logout
+        self.bugzilla.logout()
+
+    def get_bz_info(self, idlist):
+        bz_creds = credentials.get("bugzilla", None)
+
+        # login to bzapi
+        if not bz_creds:
+            raise BugzillaError("No bugzilla creds available")
```

Comment:
Please use the full form in Notifications..

Optional:
In state raise an error; I would just echo statement like "Bugzilla credentials not found" and exit
```
sys.exit(0)
```

Reply:
Hmm.. the trouble is that these commands are called during a pytest run so they will show up as INTERNALERROR, and then the code after the pytest run will try to execute, even with a system exit. I raising the BZ error is informative and shows the user why (and where) the command failed.